### PR TITLE
changed client status schema

### LINKED ISSUE #107: Client Closure status (schema change)
image

Replace enum value of NO-RESULT with UNKNOWN for the following client fields: (status_at_3_months, status_at_6_months, status_at_12_months) b/c now we are exporting stuff. Unknown better reflects that the client just has not responded.

### PR DESCRIPTION
**Pull request by ganeshasapu**
# Description

Made tweak to client closure status fields. Also apparently my IDE likes to change the way the files are formatted so it seems like I made a bunch of changes but I really just did like 4 lines of code. If anyone knows how to get rid of that lmk please.

Fixes #107 

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] I have run `npm run lint:fix`
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (if required)
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works (Delete if frontend)

### CODE REVIEW

File: server/src/models/client.model.ts [hunk 1]
```diff
@@ -47,46 +78,81 @@ export default class Client {
...
+    last_name: string;
+    assignee_id: string;
+    date_created: string;
+    status: "ACTIVE" | "INACTIVE" | "R&I" | "CLOSED";
+    phone_num: string | undefined;
+    email: string | undefined;
+    files: Array<Object> | undefined;
+    note: string | undefined;
+    closure_info:
+      | {
```
Comment (helen-li) [inline server/src/models/client.model.ts:106]: is this bar supposed to be here

File: server/src/models/client.model.ts [hunk 2]
```diff
@@ -107,32 +173,71 @@ export default class Client {
...
+      last_name: string;
+      assignee_id: string;
+      date_created: string;
+      status: "ACTIVE" | "INACTIVE" | "R&I" | "CLOSED";
+      phone_num: string | undefined;
+      email: string | undefined;
+      files: Array<Object> | undefined;
+      note: string | undefined;
+      closure_info:
+        | {
```
Comment (helen-li) [inline server/src/models/client.model.ts:189]: same with this one

**Review by kenneth-miura — APPROVED** [review]
lgtm!

File: server/src/models/client.model.ts (hunk 1 above)
Comment (ganeshasapu) [reply server/src/models/client.model.ts:106]: yeah that's weird, I'll fix that

**Review by jordanjanakievski — APPROVED** [review]
Looks good to me

**Pull request merged**